src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationParameters.java

```diff
@@ -36,21 +37,24 @@
      */
     private int maxNumberIterations;
 
+    private boolean constantPowerScaling;
+
     /**
      * Constructor with default parameters
      */
     public BalanceComputationParameters() {
-        this(DEFAULT_THRESHOLD_NET_POSITION, DEFAULT_MAX_NUMBER_ITERATIONS);
+        this(DEFAULT_THRESHOLD_NET_POSITION, DEFAULT_MAX_NUMBER_ITERATIONS, DEFAULT_CONSTANT_POWER_SCALING);
     }
 
     /**
      * Constructor with given parameters
      * @param threshold Threshold for comparing net positions (given in MW)
      * @param maxNumberIterations Maximum iteration number for balances adjustment
      */
-    public BalanceComputationParameters(double threshold, int maxNumberIterations) {
```

Comment:
When the constructor is public, we try not to delete it to prevent breaking changes. This should be kept and call the new constructor using only `DEFAULT_CONSTANT_POWER_SCALING` for the last parameter